mk/spksrc.install.mk

```diff
@@ -63,10 +68,7 @@ $(PRE_INSTALL_PLIST):
 pre_install_target: install_msg_target $(PRE_INSTALL_PLIST)
 
 install_target: $(PRE_INSTALL_TARGET)
-	$(RUN) $(MAKE) install prefix=$(INSTALL_PREFIX)
-
-install_destdir_target: $(PRE_INSTALL_TARGET)
-	$(RUN) $(MAKE) DESTDIR=$(INSTALL_DIR) install prefix=$(INSTALL_PREFIX)
+	$(RUN) $(MAKE) $(INSTALL_MAKE_OPTIONS)
```

Comment:
Was it ever using `install_target`?  By the look of the code it seemed to always go to `install_destdir_target` by default.  Not that it is important but simply curious.

Reply:
I veryfied that no package is using `install_target` so I dropped this and renamed `install_destdir_target`  to `install_target` (i.e. defined `INSTALL_MAKE_OPTIONS` taking the value of `install_destdir_target`).